modules/ui/fields/wikipedia.js

```diff
@@ -261,6 +263,26 @@ export function uiFieldWikipedia(field, context) {
     });
   }
 
+  // Fetch local language for this Wikidata ID
+  function getLocalLanguageFromWikidata(entityIDs) {
+    if (!wikidata || !entityIDs) return null;
+
+    for (let i in entityIDs) {
+      let entity = context.hasEntity(entityIDs[i]);
+      if (entity && entity.tags.wikidata) {
+        let wikidataID = entity.tags.wikidata;
+
+        let data = wikidata.itemByID(wikidataID);
+        if (data && data.sitelinks) {
+          let localLangEntry = Object.keys(data.sitelinks).find(lang => lang.endsWith('wiki'));
```

Comment:
A given Wikidata item will likely have lots of Wikipedia sitelinks (which end in `wiki`). We can’t just choose the first one; that would typically result in something like the Arabic or Azerbaijani Wikipedia getting tagged, even in a Spanish-speaking region. Please see #10789 for a discussion about possible approaches to determining the appropriate language edition to link to.